src/views/Accessibility/index.tsx

```diff
@@ -57,7 +57,13 @@ const Accessibility = () => {
   const RequestDetails = (
     <SecureRoute
       path="/508/requests/:accessibilityRequestId"
-      component={AccessibilityRequestDetailPage}
+      render={() => {
+        return (
+          <AccessibilityRequestDetailPage
+            isAccessibilityTeam={user.isAccessibilityTeam(userGroups, flags)}
```

Comment:
It might be a good idea to try to name and structure permission within the app in terms of what folks can and can't do vs. who they are:

```suggestion
            canEditTestDates={user.isAccessibilityTeam(userGroups, flags)}
```

This lets us change how we decide who can do what without needing to change a bunch of prop names at the same time. It also conceptually is a better fit for the kinds of conditionals we will be writing within components.